packages/bitcore-wallet-service/src/lib/chain/xrp/index.ts

```diff
@@ -244,6 +246,7 @@ export class XrpChain implements IChain {
     const network = tx.network;
     const unsignedTxs = tx.uncheckedSerialize();
     const signedTxs = [];
+    const txids = [];
```

Comment:
should this maybe be ` = tx.txids() || [];` so it doesn't overwrite previously added sigs?